frame/system/src/lib.rs

```diff
@@ -570,7 +570,7 @@ decl_module! {
 		/// A dispatch that will fill the block weight up to the given ratio.
 		// TODO: This should only be available for testing, rather than in general usage, but
 		// that's not possible at present (since it's within the decl_module macro).
-		#[weight = (*_ratio * T::MaximumBlockWeight::get(), DispatchClass::Operational)]
+		#[weight = *_ratio * T::MaximumBlockWeight::get()]
```

Comment:
Is this an intentional change? Moving from `Operational` to `Normal`?

Reply:
Yes, this is only used for a test and the test would require it this way. I also have an issue to remove this all-together soon https://github.com/paritytech/substrate/issues/6353